### ISSUE TITLE
Add table of neurons grouped by dissolve delay

### ISSUE THREAD

**Issue by FloorLamp** [ic-rocks/ic-rocks#12]
Create a new table or page showing a count of neurons broken down like this:

| | Dissolving | Non-dissolving |
| --- | --- | --- |
| <6 mo | 100 | 200 |
| 6mo - 1yr | 100 | 200 |
| 1yr - 2yr | 100 | 200 |
| 2yr - 3yr | 100 | 200 |
| 3yr - 4yr | 100 | 200 |
| 4yr - 8yr | 100 | 200 |

A new API endpoint will need to be added [here](https://github.com/ic-rocks/api/blob/main/routes/neurons.ts). All required data should be available for querying and grouping.

**Comment by wpb1** [comment]
We need to be able to track the history of this data.  It would be good to trend ICP in 8 yr neurons for example.

**Comment by tarek-eg** [comment]
Does count of neurons only gives a useful information? is it helpful to add sum of ICP or percentage?
for example
|  Range | Dissolving | Sum of ICP| Non-dissolving | Sum of ICP| 
| ----------------| ---------- | ------- |----------------| ---------- |
| <6 mo  | 100  |1000 |200  |2000 |
| 6mo - 1yr  | 100  |1000 |200  |2000 |
| 1yr - 2yr  | 100  |1000 |200  |2000 |
| 2yr - 3yr  | 100  |1000 |200  |2000 |
| 3yr - 4yr  | 100  |1000 |200  |2000 |
| 4yr - 8yr  | 100  |1000 |200  |2000 |

**Comment by wpb1** [comment]
Sum of ICP is the most useful metric.  Count of neuron is interesting.  If we also add percentage of ICP, then it should probably be percentage of total ICP.  If you do percentage, then probably also need to show metrics for ICP not in known neurons so the total adds up to 100% and we can see total ICP in the same chart.

**Comment by tarek-eg** [comment]
I would probably work on this one next

**Comment by wpb1** [comment]
@tarek-eg are you aware that DF defines 1 year as 365.25 days?  That will be important as you figure out how to bin the data.  Also, when people use the NNS app they have to use a slider scale to set dissolve delay.  It's not easy to be exact.

**Comment by wpb1** [comment]
hey @tarek-eg do you have any plans to work on this issue further?  If so, then can you summarize the data in tables and a chart similar to what I did in this article linked below?  I'm very hopeful to have easy access to this kind of information over in the next couple of months.  It might be a relatively large task and I would be in favor of you getting paid a portion of Norton's developer grant from dfinity for this work if he is willing to allocate some funds for it.  I would be happy to get on a conference call to discuss.

https://medium.com/@wpbartlett/the-current-state-of-governance-of-the-internet-computer-and-proposals-for-tokenomics-improvements-8d51e0a9c6eb

**Comment by tarek-eg** [comment]
Hey @wpb1,
Yes I don’t mind working on that, but I am just not sure about the availability @FloorLamp at this time to review and merge any changes.
I have already a couple of PR’s that needs to be reviewed but haven’t got any feedback yet so not sure if I should start working on something else.
